### PR TITLE
config: enable tichi for tispark

### LINKED ISSUE #530: REQUEST: deploy prow for pingcap/tispark
### GitHub Repo Name
pingcap/tispark

### Which plugins do you want to deploy
- hold
- wip
- tide
- lifecycle
- ti-community-owners
- ti-community-lgtm
- ti-community-merge
- ti-community-tars
- ti-community-label
- ti-community-label-blocker
- ti-community-cherrypicker
- ti-community-blunderbuss

### Requirements
- [x] I have carefully read the instructions for the plugin I want to use.
- [x] I agree to leave the plugin configuration to be maintained by sig-ti-community-infra.

### TODO
- [ ] @ti-community-infra/sig-community-infra  Add the requested plugin configuration
- [ ] @ti-community-infra/sig-community-infra  Add repository admin to Robots
- [ ] @ti-community-infra/sig-community-infra  Turn off the old robot function

### PR DESCRIPTION
**Pull request by Mini256**
close #530

### CODE REVIEW

File: prow/config/config.yaml [hunk 1]
```diff
@@ -1170,6 +1170,40 @@ branch-protection:
...
                 strict: true
+        tispark:
+          branches:
+            master:
+              protect: true
+              required_status_checks:
+                contexts:
+                  - 'tide'
+                  - 'license/cla'
+                  - 'idc-jenkins-ci-tispark/integration-test'
```
Comment (xuanyu66) [inline prow/config/config.yaml:1181]:
```suggestion
		  - 'idc-jenkins-ci-tispark/build'
                  - 'idc-jenkins-ci-tispark/integration-test'
```

Comment (xuanyu66) [inline prow/config/config.yaml:1181]:
```suggestion
		  - 'build'
                  - 'idc-jenkins-ci-tispark/integration-test'
```

File: prow/config/config.yaml [hunk 2]
```diff
@@ -1170,6 +1170,40 @@ branch-protection:
...
+                strict: true
+            release-2.3:
+              protect: true
+              required_status_checks:
+                contexts:
+                  - 'tide'
+                  - 'license/cla'
+                  - 'idc-jenkins-ci-tispark/build'
```
Comment (xuanyu66) [inline prow/config/config.yaml:1189]:
```suggestion
                  - 'build'
                  - 'idc-jenkins-ci-tispark/integration-test'
```

File: prow/config/config.yaml [hunk 3]
```diff
@@ -1170,6 +1170,40 @@ branch-protection:
...
+                strict: true
+            release-2.4:
+              protect: true
+              required_status_checks:
+                contexts:
+                  - 'tide'
+                  - 'license/cla'
+                  - 'idc-jenkins-ci-tispark/integration-test'
```
Comment (xuanyu66) [inline prow/config/config.yaml:1197]:
```suggestion
                  - 'build'
                  - 'idc-jenkins-ci-tispark/integration-test'
```

File: prow/config/config.yaml [hunk 4]
```diff
@@ -1170,6 +1170,40 @@ branch-protection:
...
+                strict: true
+            release-2.5:
+              protect: true
+              required_status_checks:
+                contexts:
+                  - 'tide'
+                  - 'license/cla'
+                  - 'idc-jenkins-ci-tispark/integration-test'
```
Comment (xuanyu66) [inline prow/config/config.yaml:1205]:
```suggestion
                  - 'build'
                  - 'idc-jenkins-ci-tispark/integration-test'
```

**Comment by xuanyu66** [comment]
@Mini256 PTAL

**Comment by Mini256** [comment]
/merge
/hold

Waiting for adding repository admin role for ti-chi-bot.

**Pull request merged**